internal/usecase/interfaces/scene.go

```diff
@@ -9,11 +9,21 @@ import (
 	"github.com/reearth/reearth-backend/pkg/scene"
 )
 
+type LayoutParams struct {
+	Extended *bool
+	Location,
+	NewLocation *scene.WidgetLocation
+	OldIndex,
+	NewIndex *int
+	Align *string
+}
```

Comment:
If you follow the review comments I wrote in the GraphQL schema, you will need to fix this as well.

Reply:
Yep, updated all the places affected by changes you have suggested!